Bottom: cc65e5e4d416dc45be9330f6163eef54529a2d54
Top:    8e2e5fbaa4d0da01ca4614a1eb43ad754d1e0024
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:16:20 +0000

Bump circle approximation detail level for gerber export


---

diff --git a/src/polygon.h b/src/polygon.h
index 58634f7..90d0d26 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 24 //20 //8 //40
+#define POLY_CIRC_SEGS 40 //24 //20 //8 //40
 #define POLY_CIRC_SEGS_D ((double)POLY_CIRC_SEGS)
 
 #if 0
